app-mock/src/androidTest/java/pl/dybczak/githubusers/test/UITest.kt

```diff
@@ -0,0 +1,27 @@
+package pl.dybczak.githubusers.test
+
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import pl.dybczak.githubusers.robots.SearchPageRobot
+import pl.sulej.users.view.UsersActivity
+
+@RunWith(AndroidJUnit4::class)
+class UITest {
+
+    private val searchPageRobot = SearchPageRobot()
+
+    @get:Rule
+    val activityRule = ActivityTestRule(UsersActivity::class.java)
+
+    @Test
+    fun assertUserRepoPageOpened() {
+        searchPageRobot.assertThatGitHubUsersPageDisplayed(
```

Comment:
Maybe it would be better to make those robot methods more granular and split it into multiple test cases?

```suggestion
    @Test
    fun displayLogins() {
        searchPage.assertUserDisplayed("Gorn")
        searchPage.assertUserDisplayed("Diego")
    }
    
    @Test
    fun displayRepositories() {
        searchPage.assertUserRepositoriesDisplayed("zemsta-gorna, zbroja-najemnika, new-camp")
        searchPage.assertUserRepositoriesDisplayed("old-camp, miecz-bojowy, diegos-bogen")
    }
    
    @Test
    fun filterByLogin() {
        searchPage.searchFor("Gorn")
        searchPage.assertUserDisplayed("Gorn")
        searchPage.assertUserNotDisplayed("Diego")
    }
    
    @Test
    fun filterByRepositories() {
        searchPage.searchFor("old-camp")
        searchPage.assertUserDisplayed("Diego")
        searchPage.assertUserNotDisplayed("Gorn")
    }
```